### PR TITLE
Delete accession

### PR DESCRIPTION
**Pull request by constanzauanini**
Screen Shot 2022-09-07 at 16 47 24

### CODE REVIEW

File: src/components/accession2/DeleteAccessionModal.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+export default function DelteAccessionModal(props: DelteAccessionDialogProps): JSX.Element {
+  const { onClose, open, accession } = props;
+  const history = useHistory();
+
+  const deleteHandler = async () => {
+    const response = await deleteAccession(accession.id);
+    if (response.requestSucceeded) {
+      history.push(APP_PATHS.ACCESSIONS2);
+    }
```
Comment (karthikbtf) [inline src/components/accession2/DeleteAccessionModal.tsx:25]: else toast error message

**Review by karthikbtf — APPROVED** [review]
one comment, if you want to follow up later with toast message that's fine (i will be doing error handling as a last phase)

File: src/components/accession2/DeleteAccessionModal.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/accession2/DeleteAccessionModal.tsx:25]: I will add it now

**Review by karthikbtf — APPROVED** [review]
Thanks 🎉 
I will move the toast error messaging code into a utility  soon, so it's easier to call.

**Review by karthikbtf — APPROVED** [review]

**Pull request merged**